tests/unit/test_manof.py

```diff
@@ -0,0 +1,83 @@
+import mock
+
+from twisted.internet import defer
+from twisted.trial import unittest
+
+import manof.image
+import clients.logging
+
+logger = clients.logging.TestingClient('manof_unit').logger
```

Comment:
```suggestion
logger = clients.logging.TestingClient('manof_unit_test').logger
```

Reply:
I will go further and say:
```suggestion
logger = clients.logging.TestingClient(unit_test').logger
```
Manof is redundant and contextual